src/current/v25.4/alter-database.md

```diff
@@ -355,7 +355,9 @@ For usage, see [Synopsis](#synopsis).
 
 #### Required privileges
 
-No [privileges]({% link {{ page.version.version }}/security-reference/authorization.md %}#managing-privileges) are required to reset a session setting.
+To reset default session variable values for a database with `ALTER DATABASE ... RESET {session variable}`, the user must be a member of the [`admin` role]({% link {{ page.version.version }}/security-reference/authorization.md %}#admin-role) or the [owner]({% link {{ page.version.version }}/security-reference/authorization.md %}#object-ownership) of the target database.
+
+All other session variables do not require [privileges]({% link {{ page.version.version }}/security-reference/authorization.md %}#managing-privileges) to modify.
```

Comment:
What is "other session variables" referring to here? I think we possibly could just remove this sentence entirely.

Reply:
Removed it!